recipe/meta.yaml

```diff
@@ -19,11 +19,11 @@ build:
 requirements:
   build:
     - {{ compiler('cxx') }}
-    - libtool
-    - m4
-    - autoconf   # for simpcomp and NormalizInterface
-    - automake   # for NormalizInterface
-    - git        # for NormalizInterface
+    - libtool  # FIXME: this should not be necessary for GAP itself, unless you build from git (but you don't, do you?); but perhaps some package needs it?
```

Comment:
We're building from the tarball mentioned further up.

Reply:
So that means it is *not* needed, because in release tarballs, libtool (and autoconf) were already run... At least not needed for GAP. Maybe for a package. But the whole setup is completely unclear to me by now, there are four different lists of dependencies, and I have no clue which of them are used when building packages, so I'll stop from commenting, and will trust that you know what you are doing :-)

Reply:
Let me explain. Only the first list is used when building packages. The other lists are there so that dependencies are added to runtime dependencies of those outputs.

For example, planarity is in the first list, because we build gap and all the gap packages first.
Planarity is in the 4th list because it is used as a dependency of `Digraphs` which is packaged in `gap` output. `Digraphs` is not in `gap-defaults` nor `gap-core` which means `planarity` is not in the 2nd or 3rd list.

Let me know if there's any doubt and I'll be happy to explain.

Reply:
If there was an easy way to build a subset of packages from an installed gap, we could have gotten rid of the first list and `planarity` would have appeared only in the 4th list. At the moment, gap packages need to be built at the same time using `bin/BuildPackages.sh` in the `gap` built directory.

Reply:
OK, so the first list (or set-of-lists) is *only* used for build time dependencies, while the other three are used in various runtime scenarios, got it, that makes some sense.

I am not sure why you can't build a subset of packages, though? This works well for me to build only io and cvec
```sh
cd pkg
../bin/BuildPackage.sh io* cvec*
```

And of course the four core packages gapdoc, smallgrp, transgrp, primgrp, don't need to be compiled at all.

Reply:
Thanks. I'll keep this in mind for 4.11

Btw, if you have time, have a look at https://github.com/gap-packages/JupyterKernel/pull/106